homeworks/alina.yaremenko_AlinaYaremenko/<GamesProject> /index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```

Comment:
Hey, @AlinaYaremenko, please change the name of your folder to `GamesProject` without `<>` and space at the end. https://www.mtu.edu/umc/services/websites/writing/characters-avoid/